projects/kit/src/lib/processors/with-placeholder.ts

```diff
@@ -0,0 +1,44 @@
+import {MaskitoOptions} from '@maskito/core';
+
+export function maskitoWithPlaceholder(placeholder: string): Pick<
+    Required<MaskitoOptions>,
+    'postprocessor' | 'preprocessor'
+> & {
+    removePlaceholder: (value: string) => string;
+} {
+    const removePlaceholder = (value: string): string => {
+        for (let i = value.length - 1; i >= 0; i--) {
+            const valueChar = value[i];
+            const placeholderChar = placeholder[i];
+
+            if (valueChar !== placeholderChar) {
```

Comment:
I think `value[i] !== placeholder[i]` is pretty self-explanatory without constants.

Reply:
fixed